uint16 cell_x
uint16 cell_y
float32 cell_type        # -1 = Unexplored, 0 = Obstacle, 0.5 = Goal, 1 = Covered